packages/@ember/-internals/container/lib/container.ts

```diff
@@ -403,7 +399,7 @@ function instantiateFactory(
   // SomeClass { singleton: true, instantiate: true } | { singleton: true } | { instantiate: true } | {}
   // By default majority of objects fall into this case
   if (isSingletonInstance(container, fullName, options)) {
-    let instance = (container.cache[normalizedName] = factoryManager.create());
+    let instance = (container.cache[normalizedName] = factoryManager.create() as CacheMember);
```

Comment:
Why do we need to cast here?

Reply:
I have typescript error In vscode about assign unknown property

Reply:
![image](https://user-images.githubusercontent.com/1360552/92644101-786f0700-f2eb-11ea-8275-c38cef181e50.png)